Dinah.Core/_IO/JsonFilePersister.cs

```diff
@@ -35,6 +32,8 @@ public abstract class JsonFilePersister<T> : IDisposable
 		public string Path { get; }
 		public string? JsonPath { get; }
 
+		private static Dictionary<string, object> _locks = [];
```

Comment:
I'd make this a [ConcurrentDictionary](https://learn.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentdictionary-2?view=net-9.0). Also readonly.

Reply:
Fixed